bin/libs/certificate.sh

```diff
@@ -28,11 +28,23 @@ ZWE_PRIVATE_DEFAULT_CERTIFICATE_VALIDITY="3650"
 ZWE_PRIVATE_DEFAULT_CERTIFICATE_KEY_USAGE="keyEncipherment,digitalSignature,nonRepudiation,dataEncipherment"
 ZWE_PRIVATE_DEFAULT_CERTIFICATE_EXTENDED_KEY_USAGE="clientAuth,serverAuth"
 
+
+keytool_on_path=`type keytool &> /dev/null && echo true`
+if [ "${keytool_on_path}" != "true" ]; then
+  if [ -n "${JAVA_HOME}" ]; then
+    keytool_bin="${JAVA_HOME}/bin/keytool"
+  else
+    print_message "WARNING: keytool cannot be found, please put on PATH or define JAVA_HOME"
+  fi
+else
+  keytool_bin=keytool
+fi
+
```

Comment:
This way will introduce 2 new variables and be seen by components.

Another though on this, we can put `require_java` into `zwe certificate pkcs12` sub-commands. `zwe init certificate` will call `zwe certificate` commands behind the scene. `require_java` will check JAVA_HOME and make sure it's in PATH, otherwise it will fail with error message.

What do you think?